core/domain/user_services_test.py

```diff
@@ -266,8 +272,14 @@ def test_default_identicon_data_url(self):
             'user_blue_72px.png')
         identicon_data_url = utils.convert_png_to_data_url(
             identicon_filepath)
+        # With python 2, we get new lines after every 76 characters and at the
```

Comment:
Can you clarify this comment further? The link you provide points to encodebytes but I don't see where this is being used.

Please be clear and detailed so that this can be understood by a user who has no context.

Reply:
Done

Reply:
Ah, now I get it. But I'm not sure this is the right fix. Can you instead find a function to produce the string without the '%0A' in both python 2 and python 3? In other words, we using the wrong encoding function at the moment, we should fix that (even in Python 2).

Make sure to test that this still behaves fine end-to-end.

Reply:
I am sorry I think I was unable to explain clearly. `base64.b64encode` works both in Python 2 and 3 and in both the versions, it produces the string without '%0A'. I meant encode('base64') produces string which adds '%0A'. So I am now using `base64.b64encode` in our codebase and accordingly changed the value of `DEFAULT_IDENTICON_DATA_URL`(removed '%0A'). I also removed the above comment as I changed the value of DEFAULT_IDENTICON_DATA_URL itself